util/trilateration.js

```diff
@@ -0,0 +1,126 @@
+// Created by Derrick Cohodas (dav-)
+// Based on the Python example by StackExchange user wwnick from http://gis.stackexchange.com/a/415/41129
+
+
+// Requires the Mathjs library - http://mathjs.org/
+const math = require('mathjs');
+
+/**
+ * Represents a coordinate with a distance
+ * @param {Number} lat  Latitude
+ * @param {Number} lon  Longitude
+ * @param {Number} dist Distance from [lat, lon] to some point in kilometers
```

Comment:
I don't really understand the semantic behind the distance in the beacon.
(x,y) coordinates are easy to figure out but distance... "from some point in km" ? Makes no sense to me. What point ? Is this really the best place to save this value ?